src/fuzz_introspector/utils.py

```diff
@@ -324,7 +324,11 @@ def resolve_coverage_link(
         # Handle subclass definition in the same source file
         source_file = source_file.split("$")[0]
 
-        return cov_url + os.sep + source_file + ".java.html#L" + str(lineno)
+        result = os.sep + source_file + ".java.html#L" + str(lineno)
     else:
         logger.info("Unsupported language for coverage link resolve")
-        return "#"
+
+    if result != "#":
+        result = cov_url.rstrip(os.sep) + result.replace("%s%s" % (os.sep, os.sep), os.sep)
```

Comment:
I don't think the use of `os.sep` is the way to go -- we're dealing with URLs and not filepaths. I would assume we always deal with `/` here

Reply:
Also, does it make sense to make this more general? The code as it stands does not guarantee there are no double `//`:
```
>>> s1="////".replace("//","/")
>>> s1
'//'
```